tests/testAutopas/tests/selectors/ContainerSelectorTest.cpp

```diff
@@ -1,31 +1,22 @@
-/**
- * @file ContainerSelectorTest.cpp
- * @author F. Gratl
- * @date 22.06.18
- */
-
-#include "ContainerSelectorTest.h"
-
-using ::testing::Combine;
-using ::testing::UnorderedElementsAreArray;
-using ::testing::ValuesIn;
-
-// must be TEST_F because the logger which is called in the LC constructor is part of the fixture
```

Comment:
why throw out the comment?

Reply:
Because it's the same everywhere and we use members from the ContainerSelectorTest class.